selfdrive/manager.py

```diff
@@ -440,7 +440,7 @@ def manager_thread():
   cloudlog.info({"environ": os.environ})
 
   # save boot log
-  subprocess.call(["./loggerd", "--bootlog"], cwd=os.path.join(BASEDIR, "selfdrive/loggerd"))
+  # subprocess.call(["./loggerd", "--bootlog"], cwd=os.path.join(BASEDIR, "selfdrive/loggerd"))
```

Comment:
why is this commented out?

Reply:
oh, yes. That was when loggerd didn't build yet. I'll turn that back on.